test/unittest/extension/omatadd_batched_test.cpp

```diff
@@ -0,0 +1,191 @@
+/***************************************************************************
+ *
+ *  @license
+ *  Copyright (C) Codeplay Software Limited
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  For your convenience, a copy of the License has been included in this
+ *  repository.
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  SYCL-BLAS: BLAS implementation using SYCL
+ *
+ *  @filename omatadd_batched_test.cpp
+ *
+ **************************************************************************/
+
+#include "blas_test.hpp"
+#include "extension_reference.hpp"
+
+template <typename scalar_t>
+using combination_t =
+    std::tuple<std::string, char, char, index_t, index_t, scalar_t, scalar_t,
+               index_t, index_t, index_t, index_t, index_t, index_t, index_t>;
+
+template <typename scalar_t, helper::AllocType mem_alloc>
+void run_test(const combination_t<scalar_t> combi) {
+  std::string alloc;
+  char trans_a, trans_b;
+  index_t m, n, ld_a_mul, ld_b_mul, ld_c_mul, stride_a_m, stride_b_m,
+      stride_c_m, batch_size;
+  scalar_t alpha, beta;
+
+  std::tie(alloc, trans_a, trans_b, m, n, alpha, beta, ld_a_mul, ld_b_mul,
+           ld_c_mul, stride_a_m, stride_b_m, stride_c_m, batch_size) = combi;
+
+  if ((trans_a == 'n') && (trans_b == 'n')) {
+    return;
+  }
```

Comment:
Please, could you state the reason for skipping silently the test in this case?

Reply:
@pgorlani this was actually left from the early tests as we focused mainly on the transposed cases, as the non transposed ones became trivial at a certain point in time. Will remove this skip though as it is still a case that needs to be tested!